storage/resource_mount.go

```diff
@@ -62,8 +92,8 @@ func ResourceMount() *schema.Resource {
 	tpl := GenericMount{}
 	r := commonMountResource(tpl, ResourceDatabricksMountSchema())
 	r.CreateContext = mountCallback(mountCreate).preProcess(r)
-	r.ReadContext = mountCallback(mountRead).preProcess(r)
-	r.DeleteContext = mountCallback(mountDelete).preProcess(r)
+	r.ReadContext = validateCluster(mountCallback(mountRead).preProcess(r))
+	r.DeleteContext = validateCluster(mountCallback(mountDelete).preProcess(r))
```

Comment:
Is this needed on delete as well?

Reply:
Yes. Since the resources does not have a update method, what happens instead is a "delete" and then "create".

During delete, we need to validate if the cluster is stale.